src/renderer/components/+user-management/+service-accounts/service-account-menu.tsx

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
Note: This component was extracted to separate file to avoid circular dependency